scripts/run_integration_tests.sh

```diff
@@ -78,24 +78,40 @@ popd
 
 # Build custom kernel based on virtio-pmem and virtio-fs upstream patches
 VMLINUX_IMAGE="$WORKLOADS_DIR/vmlinux"
+VMLINUX_PVH_IMAGE="$WORKLOADS_DIR/vmlinux.pvh"
 BZIMAGE_IMAGE="$WORKLOADS_DIR/bzImage"
 
-LINUX_CUSTOM_DIR="linux-custom"
+LINUX_CUSTOM_DIR="$WORKLOADS_DIR/linux-custom"
 
-if [ ! -f "$VMLINUX_IMAGE" ]; then
+if [ ! -f "$VMLINUX_IMAGE" -o ! -f "$VMLINUX_PVH_IMAGE" ]; then
     SRCDIR=$PWD
     pushd $WORKLOADS_DIR
     time git clone --depth 1 "https://github.com/cloud-hypervisor/linux.git" -b "virtio-fs-virtio-iommu-topo-5.5-rc1" $LINUX_CUSTOM_DIR
+    cp $SRCDIR/resources/linux-virtio-fs-virtio-iommu-config $LINUX_CUSTOM_DIR/.config
+    popd
+fi
+
+if [ ! -f "$VMLINUX_IMAGE" ]; then
     pushd $LINUX_CUSTOM_DIR
-    cp $SRCDIR/resources/linux-virtio-fs-virtio-iommu-config .config
+    scripts/config --disable "CONFIG_PVH"
     time make bzImage -j `nproc`
     cp vmlinux $VMLINUX_IMAGE || exit 1
     cp arch/x86/boot/bzImage $BZIMAGE_IMAGE || exit 1
     popd
-    rm -rf $LINUX_CUSTOM_DIR
+fi
+
+if [ ! -f "$VMLINUX_PVH_IMAGE" ]; then
+    pushd $LINUX_CUSTOM_DIR
+    scripts/config --enable "CONFIG_PVH"
```

Comment:
Here I would call `cp $SRCDIR/resources/linux-virtio-fs-virtio-iommu-config $LINUX_CUSTOM_DIR/.config` and then do the enabling of `CONFIG_PVH`.